modules/cluster-logging-collector-log-forward-logs-from-application-pods.adoc

```diff
@@ -23,33 +25,47 @@ spec:
   pipelines:
     - inputRefs: [ myAppLogData ] <3>
       outputRefs: [ default ] <4>
-  inputs:
+  inputs: <5>
     - name: myAppLogData
       application:
         selector:
           matchLabels:
-            environment: production <5>
-            app: nginx <5>
-        namespaces: <6>
+            environment: production <6>
+            app: nginx <6>
+        namespaces: <7>
         - app1
         - app2
-  outputs: <7>
-    - default <8>
+  outputs: <8>
+    - default <9>
```

Comment:
Based on the explanation for the `outputs` call out below, the call out `<9>` is not required as the explanation for `outputs` covers the explanation for the `default` output too.